src/create/command.ts

```diff
@@ -195,12 +222,12 @@ export async function twCreate(options: any) {
         contractPath: resolvedProjectPath,
         packageManager,
         language,
-        template,
```

Comment:
We may want to keep both template and base contract here still - base contract is for them to create a new contract from our extensions, but if we start to have contract project examples (which we probably will), they would want template to clone those as well, since they'll probably go beyond just our base contracts + extensions.